tools/notarize

```diff
@@ -39,22 +47,22 @@ fi
 if [[ "${SIGN}" == 1 && "${NOTARIZE}" == 1 ]]; then
   echo "Notarizing…"
 
-  cp LICENSE README.md "${DIR}/"
-  hdiutil create -volname "Acorn" -size 1g -srcfolder "${DIR}" -ov -format UDZO "${DMG}"
-  which gon || (go install github.com/mitchellh/gon/cmd/gon@latest)
+  which mkfs.hfsplus || apt-get install hfsprogs -y
 
-  cat <<EOF > "gon.hcl"
-notarize {
-  path = "${DMG}"
-  bundle_id = "${AC_BUNDLE}"
-  staple = true
-}
-EOF
+  # Build the DMG
+  cp LICENSE README.md "${DIR}/"
+  mkfile -n 1g "${DMG}"
```

Comment:
The size here should be based on the (sum of the) files you're going to write into it, + a bit of buffer for filesystem.

~Use `seek=(size needed) count=` instead of actually reading/writing the whole count.~ ok mkfile works the same.

The previous `hdiutil` command is making a `UDZO` = compressed read-only image, so 1GB is a mostly meaningless max size and any space you don't use compresses to ≈ nothing.  I don't know of any tools to create this kind in linux, so you'd be writing a full gig image every time here right now.